dali/test/python/test_dltensor_operator.py

```diff
@@ -153,13 +183,154 @@ def mxnet_cast(in1, in2):
     return [mxnd.cast(t, dtype='float32') for t in in1], [mxnd.cast(t, dtype='int64') for t in in2]
 
 
-def test_mxnet_flatten():
-    mxnet_case(mxnet_flatten)
+def test_mxnet_flatten_cpu():
```

Comment:
you can group all those tests like this:
```
def test_mxnet():
    for testcase in [mxnet_flatten, mxnet_slice, mxnet_cast]:
        for device in ['cpu', 'gpu']:
            yield mxnet_case, testcase, device
```

Also, please rename the file to `test_operator_dltensor.py` so that it gets automatically picked by our QA script (it looks for files named `test_operator_*.py`)

Reply:
I do not want to include this test in the `python_self_test` because it needs to import the frameworks. There is another test job `python_self_test_frameworks` which runs in L1